examples/akka-persistence-app/README.md

```diff
@@ -0,0 +1,46 @@
+## akka-persistence-app example project
+`akka-persistence-app` is an example project to show real usage of the Dump Persistence Schema Compiler Plugin. This project uses logic from the [akka-microservices-tutorial](https://developer.lightbend.com/docs/akka-platform-guide/microservices-tutorial/index.html) (the `shopping-cart-service`), with Akka Serialization Helper added.
+
+## Dump Persistence Schema Compiler Plugin usage
+You can test Dump Persistence Schema Compiler Plugin on this example project. There are two possibilities to do it:
+- `sbt compile` - compiles the project with the `dump-persistence-schema` phase enabled. `dump-persistence-schema` phase dumps detected persistence schema (types for Events and States) into `.json` files under the `target/dump-psersistence-schema-cache` directory of this project. These `.json` files are cache files used later by the `ashDumpPersistenceSchema` sbt task.
+- `sbt ashDumpPersistenceSchema` &mdash; sbt task that creates final output of the dump using mentioned `.json` files. Output is saved into the `target/akka-persistence-app-dump-persistence-schema-0.1.0-SNAPSHOT.yaml` file.
+You can just invoke `sbt ashDumpPersistenceSchema` without `sbt compile` &mdash; `ashDumpPersistenceSchema` task will invoke `sbt compile` if needed.
+
+## Running the sample code
+
+1. Prepare docker containers with:
```

Comment:
Okay... this entire setup is yours? or has it been taken from the original example?